consumer/conf/consumer/consumer.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
This should no longer be picked up from the file system. All jobs / resource should be created via API. You can make this a couple of test artifacts though and use the API to start them up there.